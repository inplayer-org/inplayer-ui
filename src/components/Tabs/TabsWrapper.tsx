import styled from 'styled-components';
import colors from 'theme/colors';

const TabsWrapper = styled.div`
  border: 1px solid ${colors.gray};
  border-radius: 3px;
  margin: 0 auto;
  display: flex;
  overflow: hidden;
  text-align: center;

  span {
    padding: 0 1rem;
  }
`;

export default TabsWrapper;
