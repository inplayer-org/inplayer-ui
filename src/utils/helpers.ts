import moment from 'moment';

export const getMonthOptions = (): Array<any> =>
  moment.months().map((label, value) => ({ value, displayName: label }));

export const getYearOptions = () => {
  const yearOptions: Array<any> = [];
  for (let i = moment().year() - 100; i <= moment().year(); i += 1) {
    yearOptions.push({ value: i.toString(), displayName: i.toString() });
  }

  return yearOptions;
};
