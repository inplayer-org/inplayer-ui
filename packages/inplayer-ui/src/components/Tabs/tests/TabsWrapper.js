import { generateSnapshots } from 'packages/inplayer-ui/src/components/Tabs/tests/helpers';

import TabsWrapper from '../TabsWrapper';

generateSnapshots(TabsWrapper);
