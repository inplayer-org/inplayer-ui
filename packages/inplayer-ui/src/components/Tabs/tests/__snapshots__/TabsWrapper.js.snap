// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsWrapper renders correctly 1`] = `
.c0 {
  border: 1px solid #d7dde5;
  border-radius: 3px;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  overflow: hidden;
  text-align: center;
}

.c0 span {
  padding: 0 1rem;
}

<div
  className="c0"
/>
`;
